test/CoinbaseSmartWallet/ExecuteWithoutChainIdValidation.t.sol

```diff
@@ -12,7 +12,7 @@ contract TestExecuteWithoutChainIdValidation is SmartWalletTestBase {
         userOpCalldata = abi.encodeWithSelector(CoinbaseSmartWallet.executeWithoutChainIdValidation.selector);
     }
 
-    function test_reverts_whenCallerNotEntryPoint() public {
+    function test_reverts_whenCallerNotENTRY_POINT() public {
```

Comment:
```suggestion
    function test_reverts_whenCallerNotEntryPoint() public {
```

Reply:
Hmm I think I may prefer to leave this. As here I am speaking of the entryPoint as a thing and not necessarily referring to our literal? Makes the test function name a bit more clunky

Reply:
oh wait that's what you said 🤦